hack/run-e2e-conformance-virtual-ocp.sh

```diff
@@ -0,0 +1,239 @@
+#!/usr/bin/env bash
+set -xeo pipefail
+
+cluster_name=${CLUSTER_NAME:-ocp-virt}
+domain_name=lab
+
+api_ip=${API_IP:-192.168.122.253}
+virtual_router_id=${VIRTUAL_ROUTER_ID:-253}
+registry="default-route-openshift-image-registry.apps.${cluster_name}.${domain_name}"
+HOME="/root"
+
+here="$(dirname "$(readlink --canonicalize "${BASH_SOURCE[0]}")")"
+root="$(readlink --canonicalize "$here/..")"
+
+check_requirements() {
+  for cmd in kcli virsh podman make go jq base64 tar; do
+    if ! command -v "$cmd" &> /dev/null; then
+      echo "$cmd is not available"
+      exit 1
+    fi
+  done
+  return 0
+}
+
+echo "## checking requirements"
+check_requirements
+echo "## delete existing cluster name $cluster_name"
+kcli delete cluster $cluster_name -y
+kcli delete network $cluster_name -y
+
+function cleanup {
+  kcli delete cluster $cluster_name -y
+  kcli delete network $cluster_name -y
+}
+
+if [ -z $SKIP_DELETE ]; then
+  trap cleanup EXIT
+fi
+
+kcli create network -c 192.168.${virtual_router_id}.0/24 --nodhcp -i $cluster_name
```

Comment:
How will it work with defined `$api_ip`?

Reply:
that is fine because the igb device is connected to an isolated network and the API is connected to a NAT network